src/App.jsx

```diff
@@ -14,33 +19,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortMethod, setSortMethod] = useState('');
+  const [reverse, setReverse] = useState('');
```

Comment:
you can just save the boolean here 
```suggestion
  const [reverse, setReverse] = useState(false);
```